secboot/secboot_sb_test.go

```diff
@@ -967,6 +967,15 @@ func (s *secbootSuite) TestResealKey(c *C) {
 			TPMPolicyAuthKeyFile: mockTPMPolicyAuthKeyFile,
 		}
 
+		numMockSealedKeyObjects := len(myParams.KeyFiles)
+		mockSealedKeyObjects := make([]*sb_tpm2.SealedKeyObject, 0, numMockSealedKeyObjects)
+		for _ = range myParams.KeyFiles {
+			mockSealedKeyFile := filepath.Join("test-data", "keyfile")
+			mockSealedKeyObject, err := sb_tpm2.ReadSealedKeyObject(mockSealedKeyFile)
+			c.Assert(err, IsNil)
+			mockSealedKeyObjects = append(mockSealedKeyObjects, mockSealedKeyObject)
+		}
+
```

Comment:
@anonymouse64 please re-review this bit. I copied a sample test keyfileobject from snapcore/secboot and now i read enough of them to satisfy all calls of keyFiles args of myParams. This way all test cases match all existing requirements. Even if in the future keyFiles is changed.